### PR TITLE
use ugbio_cnv as submodule

### PR DESCRIPTION
**Pull request by avigailmo**
This pr is the first stage of ugvc refactoring
cnv python modules were extracted to a python package in https://github.com/Ultimagen/ugbio-utils
The plan is to gradually move more and more modules to ugbio_utils and manage them as thin packages.
In the meantime - we decided to use ugbio-utils as submodule in ugvc

### CODE REVIEW

**Comment by avigailmo** [comment]
@TammushB I need your help with failed test: https://github.com/Ultimagen/VariantCalling/actions/runs/10177611939/job/28149474479

> E       AssertionError: assert False
> E        +  where False = compare_zipped_files('/tmp/pytest-of-runner/pytest-2/test_convert_cnv_results_to_vc0/EL-0059.cnv.vcf.gz', '/home/runner/work/VariantCalling/VariantCalling/ugbio_utils/src/cnv/tests/resources/test_convert_cnv_results_to_vcf/EL-0059.cnv.vcf.gz')
>

File: .github/workflows/python-package-conda.yml [hunk 1]
```diff
@@ -59,6 +63,7 @@ jobs:
           source activate genomics.py3
           python -m pytest --ignore test/unit/utils/test_db_access.py --ignore test/unit/single_cell_qc test/unit
           python -m pytest test/system
+          python -m pytest ugbio_utils/src/*/tests
```
Comment (inbalzelig-ugi) [inline .github/workflows/python-package-conda.yml:66]: I'm not sure we want to run the submodule tests on each push to variant calling (or even run submodule tests at all here)

Comment (avigailmo) [reply .github/workflows/python-package-conda.yml:66]: we do what Ilya wants 😜

Comment (ilyasoifer) [reply .github/workflows/python-package-conda.yml:66]: that's the spirit :) 
We want to be sure that ugbio modules work in the genomics.py3 environment (where we do research)

**Review by inbalzelig-ugi — APPROVED** [review]

**Review by ilyasoifer — APPROVED** [review]
@avigailmo - can you please publish instructions how to update the VariantCalling repositories once you merge?

**Pull request merged**